presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotErrorCode.java

```diff
@@ -34,6 +34,7 @@
     PINOT_DECODE_ERROR(8, EXTERNAL),
     PINOT_INVALID_PQL_GENERATED(9, INTERNAL_ERROR),
     PINOT_INVALID_CONFIGURATION(10, INTERNAL_ERROR),
+    PINOT_RETRIABLE_EXCEPTION(11, EXTERNAL, true),
```

Comment:
* My understanding is that you intend to retry on 'fetch' exceptions in the broker and segment page source codepaths. 
* If so, how about having a special exception for the 'fetch' exceptions. I don't quite like the idea of having a retriable exception, if what you really want to denote is: a fetch exception that is optionally marked as retriable.

Reply:
Yes, this is for data fetch request to pinot broker/server. Retry will happen if the query got timeout or server is not responding etc.